### ISSUE TITLE
run contact-bot throw error

### ISSUE THREAD

**Issue by lijiarui** [Chatie/wechaty#1319]
Wechaty version: 0.15.150

run `example/contact-bot.ts`, restart this code and get the following error:

```shell
18:39:44 VERB PuppetPadchat contactAvatar(D_70819983, )
18:39:44 ERR Config ###########################
18:39:44 ERR Config unhandledRejection: Error: save(/Users/egg/git/rui/wechaty/0) file is already exist! [object Promise]
18:39:44 ERR Config ###########################
18:39:44 ERR Config process.on(unhandledRejection) promise.catch(save(/Users/egg/git/rui/wechaty/0) file is already exist!)
Config Error: save(/Users/egg/git/rui/wechaty/0) file is already exist!
    at FileBox.<anonymous> (/Users/egg/git/rui/wechaty/node_modules/file-box/src/file-box.ts:383:13)
    at Generator.next (<anonymous>)
    at fulfilled (/Users/egg/git/rui/wechaty/node_modules/file-box/dist/src/file-box.js:4:58)
(node:16625) PromiseRejectionHandledWarning: Promise rejection was handled asynchronously (rejection id: 1)
18:39:55 SILL PuppetPadchatManager syncContactsAndRooms() syncing 52 out of Contact(257) & Room(113) ...
```

**Comment by zixia** [comment]
Please confirm the fix.

**Comment by lijiarui** [comment]
the error fix, but it got a new error: see more in #1332
